src/main/java/com/hackclub/hccore/slack/SlackBot.java

```diff
@@ -135,10 +210,57 @@ public SlackBot(HCCorePlugin plugin) throws Exception {
                 throw new RuntimeException(e);
               }
               return 1;
+            })).then(LiteralArgumentBuilder.<SlashCommandRequest>literal("lookup")
+            .then(RequiredArgumentBuilder.<SlashCommandRequest, String>argument("mention",
+                StringArgumentType.greedyString()).executes(context -> {
+              String mention = StringArgumentType.getString(context, "mention");
+              String id;
+              if (mention.startsWith("<@") && mention.endsWith(">")) {
+                int pipeIdx = mention.indexOf('|');
+                if (pipeIdx == -1) {
+                  id = mention.substring(2, mention.length() - 1);
+                } else {
+                  id = mention.substring(2, pipeIdx);
+                }
+              } else {
```

Comment:
could it try handling a direct user id (not in mention form)?
like if an admin wanted to lookup a minecraft account without pinging them

Reply:
yeah sure